docs/source/user_guide.md

```diff
@@ -2,64 +2,88 @@
+```
+
+By default `conda-vendor` will use a set of *virtual packages* for the
+specified platform. Under the hood `conda-vendor` uses the package
+`conda-lock`, with the default virtual packages from `conda-lock`, to solve
+the environment.[^1]  Virtual packages are the mechanism conda uses to encode
+system specific configuration.  For more information about *virtual packages*
+see the conda documentation
+[Managing virtual packages](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html).
+
+[^1]:The `conda-lock` documentation is located here:
+    [conda-lock](https://github.com/conda-incubator/conda-lock).  The set of
+default packages must be read from the `conda-lock` source code specifically
+in the `conda_lock.virtual_package` module.
+
+
+**note: If no platform option is supplied to `conda-vendor`, it will use the host platform
+as if it were supplied explicitly.  This means that `conda-vendor` will use the default
+set of *virutal packages* specified by `conda-lock`, and so the *virtual packages* might not
+be correct for the host.  To ensure a correct solution supply the virtual
+packages explictly as explaineed below.**
+
+These packages can be overwritten by providing a yaml files
+specifying the desired *virtual packages* using the `--virtual-package-spec`
+option:
+```bash
+$ conda-vendor vendor --file my_environment.yaml  \
+                      --platform linux-64 \
+                      --virtual-package-spec my_virtual_packages.yaml
+```
+
+The `conda-vendor vendor` command also has an option `--dry-run` to allow
+solving the environment without downloading files.  It outputs a json formatted
+block of text describing all the files that would be downloaded in the
+local channel.
+
+#### Using the Local channel
 
 There are several ways to use the local channel. If python was in the input `environment.yaml` file for example, the following could be used:
 
 	conda create -n test_env python -c <path_to_local_channel> --offline
-	
+
 The `--offline` flag will prevent conda from reaching out to the internet for packages. To verify that the environment created only contains packages contained in the local channel, run the following:
 
 	conda activate test_env
 	conda list --explicit
-	
-This should show a list of all the packages in the environment the local paths to their source code (typically tar.bz2 files).
 
-### Creating Environment with all Packages from Input Environment.yaml
-
-To generate a conda environment yaml that contains all the packages from the input `environment.yaml`, run the following:
-
-	conda vendor local-yaml --meta-manifest-path ./meta_manifest.yaml --channel-root <absolute_path_to_local_channel_dir>
-	
-This will create a environment file inside the `local_channel` directory called `local_conda-vendor-env.yaml`. An environment can then be created with:
+This should show a list of all the packages in the environment the local paths to their source code (typically tar.bz2 files).
 
```

Comment:
"in the environment the local paths to..." -> "in the environment with? local paths to source code"